README.md

```diff
@@ -71,7 +71,7 @@ your current trades.
 
 ### Exchange supported
 - [x] Bittrex
-- [ ] Binance
+- [x] Binance
```

Comment:
:+1:

Reply:
About Doc, could you please add a section for Binance here: https://github.com/gcarq/freqtrade/blob/develop/docs/pre-requisite.md#setup-your-bittrex-account

Reply:
Config doc: https://github.com/gcarq/freqtrade/blob/develop/docs/configuration.md

On the config, what will be Binance code? `binance`?

Reply:
done